scripts/lib/file

```diff
@@ -97,36 +98,6 @@ function create_cr_files_from_metadata() {
   done < <(echo "$alm_examples" | jq -c '.[]')
 }
 
-function create_permissions_files() {
```

Comment:
This is removed since OLM creates these automatically, correct?

Reply:
Yeah. Originally I created Roles and ClusterRoles from CSV permissions and clusterpermissions in this function but OLM creates them automatically. There should only be one OperatorGroup per namespace, so we can create one directly using the serviceAccountName in the deployment spec.